include/envoy/network/filter.h

```diff
@@ -161,6 +223,14 @@ class FilterChainFactory {
    *   false, e.g. filter chain is empty.
    */
   virtual bool createFilterChain(Connection& connection) PURE;
+
+  /**
+   * Called to create the listener filter chain.
+   * @param listener supplies the listener to create the chain on.
+   * @return true if filter chain was created successfully. Otherwise
+   *   false.
+   */
+  virtual bool createFilterChain(ListenerFilterManager& listener) PURE;
```

Comment:
`createListnerFilterChain` for this and `createNetworkFilterChain` for the one above, this shouldn't be an overloaded.

Reply:
OK.